azure.yaml

```diff
@@ -36,15 +36,15 @@ spec:
   - name: app-operator
     version: 1.0.0
   - name: azure-operator
-    version: 3.0.5-dev
+    version: 3.0.6-dev
   - name: cert-operator
     version: 0.1.0
   - name: cluster-operator
     version: 0.23.8
   - name: kubernetes
     version: 1.16.8
   - name: containerlinux
-    version: 2191.5.0
+    version: 2303.4.0
```

Comment:
This was updated in the previous release as well

Reply:
all good here, diff is funky

Reply:
Not sure I'm following here, is bumping `containerlinux` version intended for `v11.2.2`?

Reply:
Check full file, there was a bump towards 11.2.1

Reply:
oh okay there was an error in v11.2.3 that @whites11 fixed, did not notice the commit